GitUI/UserControls/ConsoleEmulatorOutputControl.cs

```diff
@@ -95,10 +120,7 @@ public override void StartProcess(string command, string arguments, string workD
                 StartupDirectory = workDir
             };
 
-            if (AppSettings.ConEmuStyle.ValueOrDefault != "Default")
-            {
-                startInfo.ConsoleProcessExtraArgs = " -new_console:P:\"" + AppSettings.ConEmuStyle.ValueOrDefault + "\"";
-            }
```

Comment:
Move this into `ConEmuControl.Start` method, modifying the signature to the following:
```cs
public ConEmuSession Start([NotNull] ConEmuStartInfo startinfo, [NotNull] JoinableTaskFactory joinableTaskFactory, [NotNull] conEmuStyle)
```
with call sites updated to:
```cs
_terminal.Start(startInfo, ThreadHelper.JoinableTaskFactory, AppSettings.ConEmuStyle.ValueOrDefault);
```

Reply:
Done but requires https://github.com/gitextensions/conemu-inside/pull/21 to be merged first...

Reply:
done